form/storage/storage_container.cpp

```diff
@@ -7,8 +7,6 @@ using namespace form::detail::experimental;
 
 Storage_Container::Storage_Container(const std::string& name) : m_name(name), m_file(nullptr) {}
 
-Storage_Container::~Storage_Container() {}
```

Comment:
Same comment as before.  Putting the implementation of the destructor in the `.cpp` makes sense.  If you want it to be the default implementation, just specify it here:

```cpp
Storage_Container::~Storage_Container() = default;
```